wallet/wallet.c

```diff
@@ -1378,19 +1385,20 @@ static void fixup_hin(struct wallet *wallet, struct htlc_in *hin)
 	if (hin->failcode || hin->failuremsg)
 		return;
 
-	hin->failcode = WIRE_TEMPORARY_CHANNEL_FAILURE;
+	hin->failcode = WIRE_TEMPORARY_NODE_FAILURE;
```

Comment:
nit: This and the next change could be added to the commit that added this fix in the first place.

Reply:
Yes, agreed.  I only realized it when I found that issue though...